src/com/theironyard/novauc/ATM.java

```diff
@@ -0,0 +1,106 @@
+package com.theironyard.novauc;
+
+import java.util.HashMap;
+import java.util.Scanner;
+
+/**
+ * Created by vtcurry on 2/27/17.
+ */
+public class ATM {
+    Scanner scanner = new Scanner(System.in);
+    HashMap<String, Double> account = new HashMap<> ();
+    String name = "";
+
```

Comment:
Awesome job Vegas.  The way you broke your methods into a different class and put them all together is very well done.